variables.tf

```diff
@@ -2,8 +2,42 @@
 # Input Variables
 ########################################################################################################################
 
-#variable "my_variable" {
-#  type        = string
-#  description = "A description of my variable"
-#  default     = "default_value"
-#}
+variable "watsonx_data_name" {
+  type        = string
+  description = "The name of the watsonx.data instance."
+}
+
+variable "resource_group_id" {
+  description = "The resource group ID where the Event Streams instance is created."
+  type        = string
+}
+
+variable "region" {
+  default     = "us-south"
+  description = "The region that's used with the IBM Cloud Terraform IBM provider. It's also used during resource creation."
+  type        = string
+  validation {
+    condition     = contains(["eu-de", "us-south", "eu-gb", "jp-tok"], var.region)
+    error_message = "You must specify `eu-de`, `eu-gb`, `jp-tok` or `us-south` as the IBM Cloud region."
+  }
+}
+
+variable "existing_data_instance" {
+  default     = null
+  description = "CRN of the an existing watsonx.data instance."
+  type        = string
+}
+
+variable "watsonx_data_plan" {
+  default     = "do not install"
+  description = "The plan that's used to provision the watsonx.data instance."
+  type        = string
+  validation {
+    condition = anytrue([
+      var.watsonx_data_plan == "do not install",
```

Comment:
I think we should support `do not install` here

Reply:
I don't see any `do not install` plan in the [documentation](https://cloud.ibm.com/docs/watsonxdata?topic=watsonxdata-getting-started) so why are we adding this in the module?

Reply:
sorry I meant to say 
I think we should **not** support do not install here